ramani-maplibre/src/main/java/org/ramani/compose/Polyline.kt

```diff
@@ -16,6 +16,16 @@ import androidx.compose.runtime.currentComposer
 import org.maplibre.android.geometry.LatLng
 import org.maplibre.android.plugins.annotation.LineOptions
 
+enum class PolylineDashType(val dashArray: Array<Float>?) {
+    NoDash(null),
+    Dash(arrayOf(2f, 1f)),
+    LongDash(arrayOf(3f, 1f));
+    
+    companion object {
+        fun fromInt(value: Int) = entries.firstOrNull { it.ordinal == value } ?: NoDash
+    }
+}
+
```

Comment:
Hmm I don't know what I think about this. I am fine changing `isDashed: Boolean` below with `dashType: Array<Float>?`, but I wonder if we should not leave the rest to the user.

I am not completely convinced that those 10 lines are worth it. Maybe a documentation comment on `Polyline` that would explain the different parameters and would suggest using those values for "dash" and "long dash"? What do you think?

Reply:
The user can still define their own dash array, but I thought it might be nice to have some established dash types to easily pick from. Do you want me to just change the default value to null and take out this enum?

Reply:
> Do you want me to just change the default value to null and take out this enum?

Yes, I think so. This will keep the default behavior (not dashed) and still expose the dash pattern 👍